hadoop-ozone/dist/src/main/smoketest/s3/MultipartUpload.robot

```diff
@@ -54,16 +52,14 @@ Test Multipart Upload
 # upload we get error entity too small. So, considering further complete
 # multipart upload, uploading each part as 5MB file, exception is for last part
 
-	${system} =         Evaluate    platform.system()    platform
-	Run Keyword if      '${system}' == 'Darwin'  Create Random file for mac
-	Run Keyword if      '${system}' == 'Linux'   Create Random file for linux
-	${result} =         Execute AWSS3APICli     upload-part --bucket ${BUCKET} --key multipartKey --part-number 1 --body /tmp/part1 --upload-id ${nextUploadID}
-	                    Should contain          ${result}    ETag
+    ${system} =         Evaluate    platform.system()    platform
```

Comment:
We can remove line 55, as we have previously checked system type and created random file. 
Now it is removed, we don't need it.